core/server/master/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -165,27 +166,24 @@ public Object getFieldValue(MasterWorkerInfo o) {
   /**
    * Creates a new instance of {@link BlockMaster}.
    *
-   * @param registry the master registry
    * @param journalFactory the factory for the journal to use for tracking master operations
    */
-  public BlockMaster(MasterRegistry registry, JournalFactory journalFactory) {
-    this(registry, journalFactory, new SystemClock(), ExecutorServiceFactories
+  protected BlockMaster(JournalFactory journalFactory) {
```

Comment:
Protected usually means that it's used by a subclass, for this usage I think package-private (no modifier) is be more correct.